docs/manual/manual_buzzer_control.rst

```diff
@@ -0,0 +1,183 @@
+
+If your module plays the watchdog error song, please reach out to our support team at support@vertiq.co with the details surrounding the watchdog. 
+Any flight logs, support files, etc. that can be provided will help our team best diagnose and fix the issue in a new firmware release.
+
+Buzzer Song Volume
+====================
+If the audio feedback from your module is either too loud or quiet, you can adjust its volume with the Volume Max parameter available in the Control Center's advanced tab.
+
+.. image:: ../_static/manual_images/buzzer_song_mp3s_pics/volume_max.png
+
+To increase the volume, increase the maximum voltage, to decrease the volume, decrease maximum voltage.
+
+Custom Songs with the Buzzer Control Client
+===================================================
+Since the buzzer controller is a :ref:`standard IQUART client <buzzer_control>`, you can communicate with it via IQUART. The examples presented here will be performed using the :ref:`Python API <getting_started_python_api>`.
+
+In order to play a note of your choosing, you must set, in order, a frequency, duration, and volume. Only then can you trigger playback.
+
+Suppose you want your module to play a middle C (261.6 Hz) for 3 seconds. 
+
+.. code-block:: python
+
+    import iqmotion as iq
+    import time
+
+    # Module Communication - This must be updated with the correct parameters for your module's serial communication
+    com = iq.SerialCommunicator("COM3", baudrate=115200)
+
+    # Using the Vertiq4006 with additional custom client files
+    module = iq.Vertiq2306(com, firmware="speed")
+
+    #Play a middle C for 3 seconds
+    module.set("buzzer_control", "hz", 261.6)
+    module.set("buzzer_control", "duration", 3000)
+    module.set("buzzer_control", "volume", 100)
+    module.set("buzzer_control", "ctrl_note")
+
+    #Let it finish
+    time.sleep(5)
+
+    #Release the module
+    module.set("buzzer_control", "ctrl_coast")
+
+While the module is playing a note, its reported ``ctrl_mode`` is note (2). This value is polled in order to expand our basic application to play basic songs. 
+The following can be used as a basis to create custom songs, and provides an example of playing the C-Major scale.
+
+.. code-block:: python
+
+    import iqmotion as iq
+    import numpy as np
+
+    # Module Communication - This must be updated with the correct parameters for your module's serial communication
+    com = iq.SerialCommunicator("COM3", baudrate=115200)
+
+    # Using the Vertiq2306 with additional custom client files
+    module = iq.Vertiq2306(com, firmware="speed")
+
+    #Information about how loud to make each note as well as the frequency of each note in the C-Major scale
+    volume = 100
+    c = 261.6
+    d = 293.7
+    e = 329.7
+    f = 349.2
+    g = 392
+    a = 440
+    b = 493
+
+    #Fill in our song information
+    duration = 750 #ms
+    c_scale = [c, d, e, f, g, a, b, c*2]
+    c_scale_durations = np.full(len(c_scale), duration)
```

Comment:
It doesn't matter, but you could get what I believe is the same array without numpy by doing [duration]*len(c_scale). Gives you a list that is just duration c_scale times, like [750, 750, 750, 750, 750, 750, 750]. Not really sure if that is actually better really, just thought I'd mention there's a "base python" way to do this too